backend/prismaCalls.ts

```diff
@@ -351,4 +351,98 @@ export async function getS3Link(metadataId: string): Promise<string | null> {
         logger.error(`Error in getS3Link: ${error}`);
         throw error;
     }
-}
\ No newline at end of file
+}
+
+export async function checkPackageExistsID(packageID: string): Promise<boolean> {
```

Comment:
There's already a checkPackageExists function. Just make the 2 arguments in checkPackageExists optional like the 3rd one is and then use that instead of making 2 new functions.